--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -168,9 +168,9 @@
 
 ## Add the install directives for the runtime library.
 install(TARGETS ${ROCM_SMI_TARGET}
-          LIBRARY DESTINATION ${ROCM_SMI}/lib COMPONENT ${ROCM_SMI_COMPONENT})
+          LIBRARY DESTINATION lib COMPONENT ${ROCM_SMI_COMPONENT})
 install(FILES ${SOURCE_DIR}/include/rocm_smi/rocm_smi.h
-                                        DESTINATION rocm_smi/include/rocm_smi)
+                                        DESTINATION include/rocm_smi)
 
 ## Add the packaging directives for the runtime library.
 
