---
 CMakeLists.txt |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -194,11 +194,11 @@ install ( FILES ${CMAKE_CURRENT_BINARY_D
 
 ## Add the install directives for the runtime library.
 install(TARGETS ${ROCM_SMI_TARGET}
-          LIBRARY DESTINATION ${ROCM_SMI}/lib COMPONENT ${ROCM_SMI_COMPONENT})
+          LIBRARY DESTINATION lib COMPONENT ${ROCM_SMI_COMPONENT})
 install(FILES ${SOURCE_DIR}/include/rocm_smi/rocm_smi.h
-                                        DESTINATION rocm_smi/include/rocm_smi)
+                                        DESTINATION include/rocm_smi)
 install(FILES ${SOURCE_DIR}/include/rocm_smi/kfd_ioctl.h
-                                        DESTINATION rocm_smi/include/rocm_smi)
+                                        DESTINATION include/rocm_smi)
 
 # Generate Doxygen documentation
 find_package(Doxygen)
